workflow/notebooks/cohort-page.ipynb

```diff
@@ -329,7 +339,8 @@
     "        sizing_mode='stretch_width', \n",
     "        show=False, \n",
     "        width=800, \n",
-    "        genes_height=100\n",
+    "        track_height=150,\n",
+    "        genes_height=90,\n",
```

Comment:
I tweaked the height of the GWSS tracks to make things more compact, so it's easier to make a visual connection between the H12 track and the genes track at the bottom.